CHANGELOG.md

```diff
@@ -0,0 +1,68 @@
+# Change Log
```

Comment:
Yep, I would encourage contributors to add a summary of the contents of their pull requests to the CHANGELOG file. It could even be used to post on the website a monthly summary of the changes so everybody is updated :)